pkgs/applications/networking/cluster/karmor/default.nix

```
@@ -0,0 +1,36 @@
+{ buildGoModule, fetchFromGitHub, installShellFiles, lib }:
+
+buildGoModule rec {
+  pname = "karmor";
+  version = "0.11.1";
+
+  src = fetchFromGitHub {
+    owner = "kubearmor";
+    repo = "kubearmor-client";
+    rev = "v${version}";
+    hash = "sha256-s1G5ZcXtjL9TxYpEUvnqiQXaY7OFUwCDXFongRM48Xk=";
+  };
+
+  vendorHash = "sha256-VvjcGiBxK2OVvIEc/ScwUT6zJZTccnXu/JfXKXc5WNY=";
+
+  nativeBuildInputs = [installShellFiles];
+
+  # integration tests require network access
+  doCheck = false;
+
+  postInstall = ''
+    mv $out/bin/{${src.repo},${pname}}
```

```suggestion
    mv $out/bin/{kubearmor-client,karmor}
```
or maybe symlink it instead. Also below we should not substitute pname

---

this is the binary name that they use upstream, so leaving `kubearmor-client` will likely only confuse consumers; also, my use of `${pname}` here is to prevent typos and dyslexia from creating subtle bugs. feel free to suggest changes, I will happily accept them, but I do not trust myself type the same string correctly every time.